internal/pkg/term/selector/selector.go

```diff
@@ -106,6 +125,14 @@ func NewDeploySelect(prompt Prompter, configStore AppEnvLister, deployStore Depl
 	}
 }
 
+// NewEc2Select returns a new selector that chooses Ec2 resources.
+func NewEc2Select(prompt Prompter, ec2Client VPCSubnetLister) *Ec2Select {
```

Comment:
nit: https://github.com/golang/go/wiki/CodeReviewComments#initialisms
```suggestion
func NewEC2Select(prompt Prompter, ec2Client VPCSubnetLister) *EC2Select {
```